### DISCUSSION TITLE
deploy supabase docker locally, rest api response chinese content

### DISCUSSION THREAD

**Discussion by wangweiji1006** [supabase/supabase#26320] [Questions]
deploy supabase docker locally, I hope to be able to return Chinese content in the 'message' or 'details' data responded to by the rest API.
```
import { createClient } from '@supabase/supabase-js'

const supabase = createClient('http://127.0.0.1:54321', 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24iLCJleHAiOjE5ODM4MTI5OTZ9.CRXP1A7WOeoJeXxjNni43kdQwgnWNReilDMblYTn_I0')

const { data, error } = await supabase
  .from('t_student')
  .insert([
    { name: 'someValue' },
  ])
  .select()

  console.log(data, error)
```
error return :
```
{
  code: '42501',
  details: null,
  hint: null,
  message: 'new row violates row-level security policy for table "t_student"'
}
```
Question1: Can we support returning to Chinese by details or message?
Question2: If Question1 cannot be achieved, can you provide a complete list of code values for our self translation of Chinese?

**Comment by silentworks** [comment]
These messages come directly from Postgres from what I can tell so you would have to figure out how to change this in Postgres itself. Same for question 2, you would have to check Postgres for all its potential error codes and also PostgREST.
